<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="testkwshibb1.nerc.ac.uk" name="uk.ac.cardiff.raptormua.engine.statistics.StatisticProcessorFactoryTest" tests="1" failures="1" timestamp="1 Dec 2011 18:50:47 GMT" time="0.005" errors="0">
  <testcase name="getProcessorTest" time="0.005" classname="uk.ac.cardiff.raptormua.engine.statistics.StatisticProcessorFactoryTest">
    <failure type="uk.ac.cardiff.raptormua.engine.statistics.StatisticPostProcessorFactoryException" message="uk.ac.cardiff.raptormua.engine.statistics.ProcessorRegistryException: No such processor with type test">
      <![CDATA[uk.ac.cardiff.raptormua.engine.statistics.StatisticPostProcessorFactoryException: uk.ac.cardiff.raptormua.engine.statistics.ProcessorRegistryException: No such processor with type test
	at uk.ac.cardiff.raptormua.engine.statistics.StatisticProcessorFactory.getPostProcessor(StatisticProcessorFactory.java:56)
	at uk.ac.cardiff.raptormua.engine.statistics.StatisticProcessorFactoryTest.getProcessorTest(StatisticProcessorFactoryTest.java:27)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:39)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
	at java.lang.reflect.Method.invoke(Method.java:597)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:76)
	at org.testng.internal.MethodInvocationHelper$1.runTestMethod(MethodInvocationHelper.java:178)
	at org.springframework.test.context.testng.AbstractTestNGSpringContextTests.run(AbstractTestNGSpringContextTests.java:158)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:39)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
	at java.lang.reflect.Method.invoke(Method.java:597)
	at org.testng.internal.MethodInvocationHelper.invokeHookable(MethodInvocationHelper.java:191)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:666)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:846)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1170)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.runWorkers(TestRunner.java:1147)
	at org.testng.TestRunner.privateRun(TestRunner.java:749)
	at org.testng.TestRunner.run(TestRunner.java:600)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:317)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:312)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:274)
	at org.testng.SuiteRunner.run(SuiteRunner.java:223)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1039)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:964)
	at org.testng.TestNG.run(TestNG.java:900)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:110)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:205)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:174)
Caused by: uk.ac.cardiff.raptormua.engine.statistics.ProcessorRegistryException: No such processor with type test
	at uk.ac.cardiff.raptormua.engine.statistics.DefaultStatisticProcessorRegistry.getProcessor(DefaultStatisticProcessorRegistry.java:33)
	at uk.ac.cardiff.raptormua.engine.statistics.StatisticProcessorFactory.getPostProcessor(StatisticProcessorFactory.java:47)
	... 33 more
]]>
    </failure>
  </testcase>
</testsuite>
